docs/source/commandline/commandline-all.rst

```diff
@@ -928,95 +1148,176 @@ Core Video Options
+    case you should use **-notriplebuffer -waitvsync**. Note that this option
+    does not work with **-video gdi** mode.
+
+    The default is OFF (**-nowaitvsync**).
 
-	Note that SDL-based MAME support for this option depends entirely on your operating system and video drivers; in general it will not work in windowed mode so **-video opengl** and fullscreen give the greatest chance of success.
+    Note that SDL-based MAME support for this option depends entirely on your
+    operating system and video drivers; in general it will not work in windowed
+    mode so **-video opengl** and fullscreen give the greatest chance of
+    success.
 
 .. _mame-commandline-syncrefresh:
 
 **-[no]syncrefresh**
 
-	Enables speed throttling only to the refresh of your monitor. This means that the system's actual refresh rate is ignored; however, the sound code still attempts to keep up with the system's original refresh rate, so you may encounter sound problems. This option is intended mainly for those who have tweaked their video card's settings to provide carefully matched refresh rate options. Note that this option does not work with -video gdi mode. The default is OFF (**-nosyncrefresh**).
+    Enables speed throttling only to the refresh of your monitor.  This means
+    that the system's actual refresh rate is ignored; however, the sound code
+    still attempts to keep up with the system's original refresh rate, so you
+    may encounter sound problems.
+
+    This option is intended mainly for those who have tweaked their video card's
+    settings to provide carefully matched refresh rate options.  Note that this
+    option does not work with **-video gdi** mode.
+
+    The default is OFF (**-nosyncrefresh**).
 
 .. _mame-commandline-prescale:
 
 **-prescale** *<amount>*
 
-	Controls the size of the screen images when they are passed off to the graphics system for scaling. At the minimum setting of 1, the screen is rendered at its original resolution before being scaled. At higher settings, the screen is expanded by a factor of *<amount>* before being scaled. With **-video d3d**, this produces a less blurry image at the expense of some speed. The default is ``1``.
+    Controls the size of the screen images when they are passed off to the
+    graphics system for scaling.  At the minimum setting of 1, the screen is
+    rendered at its original resolution before being scaled.  At higher
+    settings, the screen is expanded by a factor of *<amount>* before being
+    scaled.  With **-video d3d**, this produces a less blurry image at the
+    expense of some speed.
 
-	This is supported with all video output types (``bgfx``, ``d3d``, etc) on Windows and is **ONLY** supported with OpenGL on other platforms.
+    The default is ``1``.
+
+    This is supported with all video output types (``bgfx``, ``d3d``, etc) on
+    Windows and is **ONLY** supported with OpenGL on other platforms.
```

Comment:
It's worth pointing out that this scales the the video by this amount in both axes using nearest-neighbour sampling *before* applying filtering and/or shaders (but in a less uncomfortable sentence structure than this comment).

Reply:
Okay. We'll see if you like the new version of this.